portfolio/src/main/webapp/script.js

```diff
@@ -93,12 +96,34 @@ async function getMessages() {
 }
 
 async function getComments() {
-  const response = await fetch("/comments");
+  const numComments = document.getElementById("user-result").value;
+  const response = await fetch("/comments?user-choice=" + numComments);
   const comment = await response.json();
   var comments = "";
-  console.log(comment);
+//   console.log(comment);
   for (var i = 0; i < comment.length; i++) {
     comments += "-" + comment[i].text + "\n";
   }
   document.getElementById("comment-container").innerText = comments;
 }
+
+async function getAllComments(){
+    const response = await fetch("/comments-list");
+    const comment = await response.json();
+    console.log(comment);
+    for(var i=0; i<comment.length; i++){
+        idArray.push(comment[i].id);
+    }
+    console.log(idArray);
+    return idArray;
+}
+
+console.log(idArray);
+
+async function deleteAll(){
+    for(var i=0; i<idArray.length ; i++){
+        const params = new URLSearchParams();
+        params.append('id', idArray[i]);
+        fetch('/delete-data', {method: 'POST', body: params});
```

Comment:
We'll let this go for now, but normally you want to make as few HTTP calls as possible. E.g. make a single HTTP delete call which includes a list of IDs rather than making N HTTP separate delete calls.